pds/transactions/exampleNFT/link_providerCap_exampleNFT.cdc

```diff
@@ -0,0 +1,24 @@
+import NonFungibleToken from "NonFungibleToken"
+import ExampleNFT from "ExampleNFT"
+import MetadataViews from "MetadataViews"
+
+transaction (nftWithdrawCapPath: StoragePath) {
```

Comment:
is this tx gonna be needed to run now?

Reply:
right we need to run it prior to running `create_distribution` (same as before)

if that's easier, tho, we could add a fallback in `create_distribution` to create a capability on the spot if it doesn't find the `withdrawCap` in account storage - one downside is that it might be a bit harder to keep track of the different places we create and use withdraw capabilities

Reply:
Cool! we can get this in first then i will update the transactions in platform services